app-backup/deja-dup/deja-dup-40.6.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit meson eutils gnome2-utils vala xdg
```

Comment:
Sort when possible. Also since you're overriding all the phases, `xdg-utils`, not `xdg`. Where are `eutils` used?